packages/paste-website/src/pages/components/detail-text/index.mdx

```diff
@@ -0,0 +1,219 @@
+  storybookUrl="/?path=/story/components-detail-text"
+  data={props.data}
+/>
+
+---
+
+<contentwrapper>
+
+<PageAside data={props.data.mdx} />
+
+<content>
+
+<LivePreview scope={{DetailText, Box}} noInline>
+  {detailTextWithImage}
+</LivePreview>
+
+## Guidelines
+
+### About Detail Text
+
+The Detail Text component is used for small pieces of text that are secondary to body text in the hierarchy of a page. Common examples include an image caption, chart element legend, or a paragraph footnote.
+
+Detail Text should not be used for body text, and should be reserved for small pieces of content only. It should not be used solely for its stylistic properties within other typographic elements.
+
+### Accessibility
+
+The Detail Text component is a `div` element by default. You can modify the element in situations where a `div` isn’t semantically correct. For example, set `as=”figcaption”` if working with Detail Text inside a [`figure` element](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/figure).
+
+If your Detail Text is used to describe another element, use an `id` for the Detail Text component and `aria-describedby` for the other element to make the connection clear.
+
+<Callout variant="neutral">
+  <CalloutText>
+    Detail Text is not the same as alt text on images. The context and content of your usage might mean including alt
+    text for further detail, or making alt text an empty string to prevent redundancy.{' '}
+    <Anchor href="https://www.w3.org/WAI/tutorials/images/decision-tree/" showExternal>
+      Learn more about alt text and when to use it.
+    </Anchor>
+  </CalloutText>
+</Callout>
+
+<Box paddingBottom="space70" />
+
+---
+
+## Examples
+
+### Detail Text below an image
+
+Use Detail Text to add a caption below an image.
+
+<LivePreview scope={{DetailText, Box}} noInline>
+  {detailTextWithImage}
+</LivePreview>
+
+### Detail Text below primary content
+
+Use Detail Text to add a description below primary content for extra context.
+
+<LivePreview scope={{DetailText, Table, THead, Tr, Th, Td, TBody, Text}} noInline>
+  {detailTextWithPrimaryContent}
+</LivePreview>
+
+### Detail Text with no top margin
+
+Detail Text includes a marginTop prop to remove its default top margin of `space30`.
+
+<LivePreview scope={{DetailText, Text, Box, Heading, ArrowUpIcon, Card}} noInline>
+  {detailTextWithNoMargin}
+</LivePreview>
+
+### Detail Text as a footnote
+
+Use Detail Text to add a footnote below a paragraph.
+
+<LivePreview scope={{DetailText, Heading, Paragraph}} noInline>
+  {detailTextAsFootnote}
+</LivePreview>
+
+---
+
+## When to use Detail Text
+
+<DoDont>
+  <Do
+    title="Do"
+    body="Use Detail Text for small pieces of text that are not the main content of the UI, like graph legends and image captions"
+  />
+  <Dont title="Don't" body="Don’t use Detail Text for body content. Consider using Paragraph instead." />
```

Comment:
Add link to https://paste.twilio.design/components/paragraph/

Reply:
We can't add links to Do/Don't blocks, they dont accept MDX.